tests/instruments/tcpowerconversion/test_cxn.py

```diff
@@ -130,4 +130,4 @@ def test_load_capacity_preset(channel):
             (cmd, b"\x2a" + response),
         ],
     ) as inst:
-        assert inst.presets[channel].load_capacity == 50
+        assert inst.channels[channel].load_capacity == 50
```

Comment:
Your changes are good, but we have to take care, that we do not break the instruments for the users, probably the original author uses the driver.

Reply:
Same as above re: breaking changes